keda/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/kedacore/keda:1.2.0
+FROM docker.io/kedacore/keda:master
```

Comment:
Let's pin it

Reply:
@andriisoldatenko @dimberman what do you guys think here?

Reply:
oh wait, outdated.. nvm